src/sentry/conf/server.py

```diff
@@ -726,6 +734,11 @@ def create_partitioned_queues(name):
         "schedule": timedelta(hours=1),
         "options": {"expires": 3600, "queue": "incidents"},
     },
+    "monitor-release-adoption": {
+        "task": "sentry.tasks.monitor_release_adoption",
+        "schedule": timedelta(hours=1),
```

Comment:
Does it matter when this runs each hour? It could make sense to use `crontab` to specify which minute it runs, so that it always happens an hour apart.

Reply:
Yeah I think the consistency would be good. Changed it to `crontab(minute=0)`